src/Compiler.php

```diff
@@ -197,10 +198,24 @@ protected function getAttributesFromAttributeString(string $attributeString): ar
             } else {
                 $value = str_replace("'", "\\'", $value);
                 $value = "'{$value}'";
+
+                if (Str::contains($attribute, ':')) {
+                    $namespace = Str::before($attribute, ':');
+                    if (! $namespaces->has($namespace)) {
+                        $namespaces->put($namespace, collect());
+                    }
+
+                    $attribute = Str::after($attribute, ':');
+                    $namespaces[$namespace]->put($attribute, $value);
```

Comment:
Maybe we can use `data_set($namespaces, "{$namespace}.{$attribute}", $value)` here (`$namespaces` will have to be an array). That'll save us from having to check if the namespace key has been set before :)

Reply:
Good suggestion, I'll take care of that before tagging a new release.